index.html

```diff
@@ -23,9 +23,13 @@
         
         <script type="text/javascript" src="javascript/custom/ng-currency.js"></script>
         
-        <script type="text/javascript" src="javascript/retirement_calculator_module.js"></script>
-        <script type="text/javascript" src="javascript/create_retirement_graph_module.js"></script>
         <script type="text/javascript" src="javascript/app.js"></script>
+        <script type="text/javascript" src="javascript/date_service.js"></script>
+        <script type="text/javascript" src="javascript/interest_service.js"></script>
+        <script type="text/javascript" src="javascript/geometry_service.js"></script>
+        <script type="text/javascript" src="javascript/retirement_calculator_service.js"></script>
+        <script type="text/javascript" src="javascript/create_retirement_graph_service.js"></script>
+        

```

Comment:
Is index.html just going to grow in size for each new service we create? No biggie, but feels like there's a way to assemble all the pieces at build-time, with Grunt or something